src/ast/codegen_llvm.cpp

```diff
@@ -86,10 +86,15 @@ void CodegenLLVM::visit(Builtin &builtin)
   }
   else if (builtin.ident == "elapsed")
   {
-    struct timespec ts;
-    // this time will be calculated during codegen and baked into the BPF
-    clock_gettime(CLOCK_MONOTONIC, &ts);
-    expr_ = b_.CreateSub(b_.CreateGetNs(), b_.getInt64(1000000000ULL * ts.tv_sec + ts.tv_nsec));
+    AllocaInst *key = b_.CreateAllocaBPF(b_.getInt64Ty(), "key");
```

Comment:
Could you give this a more descriptive name, e.g. "elapsed_key"? It would make the LLVM IR easier to follow